client/source/utilities/axiosRequests.js

```diff
@@ -26,7 +26,7 @@ const get = function getByEndpoint(endpoint) {
  * */
 const post = function postBygetByEndpoint(endpoint, json) {
```

Comment:
Very small issue that probably doesn't matter, but should it be postByGetByEndpoint to include 'g' in camel-casing?

Reply:
It shouldn't even be like that, VSCode replace didn't repalce as smoothly as I wanted it to!